jha/src/views/enrolment/FormSelectionPage.vue

```diff
@@ -85,6 +89,11 @@ export default {
       applicationUuid: null,
     };
   },
+  computed: mapGetters({
+    isEligibleForMSP: "enrolmentModule/isEligibleForMSP",
+    isEligibleForFPCare: "enrolmentModule/isEligibleForFPCare",
+    isEligibleForSuppBen: "enrolmentModule/isEligibleForSuppBen",
```

Comment:
Can you use the module constant for these getters? Just for in case we have to change the string value for whatever reason.

Example:
```javascript
isEligibleForMSP: `${enrolmentModule}/isEligibleForMSP`
```

Reply:
Done